src/pages/Product/Product.js

```diff
@@ -2,8 +2,111 @@ import React, { Component } from 'react';
 import './Product.scss';
 
 class Product extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isProductListOn: false,
+      isScentsListOn: false,
+    };
+  }
+
+  toggleProductList = () => {
+    this.setState({
+      isProductListOn: !this.state.isProductListOn,
+    });
+  };
+
+  toggleScentsList = () => {
+    this.setState({
+      isScentsListOn: !this.state.isScentsListOn,
+    });
+  };
+
   render() {
-    return <div></div>;
+    console.log(this.state);
+    return (
+      <div className="contents">
+        <div className="conbox">
```

Comment:
```suggestion
        <div className="contentBox">
```
이정도는 그냥 명확하게 적어주는게 좋을거 같아요

Reply:
네!!